testsuite/shared/src/main/java/org/jboss/as/test/integration/management/cli/CliProcessWrapper.java

```diff
@@ -203,7 +205,7 @@ private void pushToInput(String line) {
         }
     }
 
-    private int resultTimeout = 10000;
+    private int resultTimeout = TimeoutUtil.adjust(30000);
```

Comment:
@jfdenise Any concerns with this? Best practice is to use TimeoutUtil.adjust. I don't know if a 30 second base value means allowing incorrect behavior a test should catch. My impression is not, but I wanted to check with you. Taking a longer time to detect a failure is fine; that's a good tradeoff for avoiding false failures due to things like long gc pauses of CI slave VMs being unscheduled under load on the machine.